src/components/sidebar/treeNode.tsx

```diff
@@ -213,10 +213,28 @@ const TreeNode = ({
     }
   }, [isCollapsed])
 
+  React.useEffect(() => {
```

Comment:
Wouldn't it be better to create an array of these special cases and keep it in `config.ts` so that we can add more special cases (if any) in the future? 
ex: 
```
const specialPaths = ['prisma-cli', 'deployment', 'sql-views']
```
then you can iterate over this array for `const specialCases`

Reply:
reviewed! thanks for the tip :)